Author: Steve Langasek <vorlon@debian.org>
Description: update our dbus policy to match that from systemd 215
Bug-Debian: http://bugs.debian.org/765101

Index: debian/data/org.freedesktop.systemd1.conf
===================================================================
--- debian.orig/data/org.freedesktop.systemd1.conf
+++ debian/data/org.freedesktop.systemd1.conf
@@ -64,6 +64,10 @@
 
                 <allow send_destination="org.freedesktop.systemd1"
                        send_interface="org.freedesktop.systemd1.Manager"
+                       send_member="ListUnitsFiltered"/>
+
+                <allow send_destination="org.freedesktop.systemd1"
+                       send_interface="org.freedesktop.systemd1.Manager"
                        send_member="ListUnitFiles"/>
 
                 <allow send_destination="org.freedesktop.systemd1"
@@ -87,6 +91,10 @@
                        send_member="Dump"/>
 
                 <allow send_destination="org.freedesktop.systemd1"
+                       send_interface="org.freedesktop.systemd1.Manager"
+                       send_member="GetDefaultTarget"/>
+
+                <allow send_destination="org.freedesktop.systemd1"
                        send_interface="org.freedesktop.systemd1.Scope"
                        send_member="Abandon"/>
 
